server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/aggregators/MetricEntry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.compositeindex.datacube.startree.aggregators;
+
+import org.opensearch.index.compositeindex.datacube.MetricStat;
+
+/**
+ * Holds the pair of metric name and it's associated stat
+ *
+ * @opensearch.experimental
+ */
+public class MetricEntry {
```

Comment:
Lets move it to meta package or somewhere similar since its only used in file formats